packages/itwinui-variables/mixins.scss

```diff
@@ -0,0 +1,275 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@mixin light-variables {
+  --iui-color-background-1-rgb: 255, 255, 255;
```

Comment:
Not hsl?

Reply:
Also, are those `*-rgb` still needed?

Reply:
i copied these from our current generated css. I can convert to hsl I suppose

the `-rgb`/`-hsl` would allow easily changing the opacity if needed

Reply:
As I mentoined, having separate `rgb` variable looks weird.. WOuld you mind dropping some examples from other products, which you mentioned have the same?

Reply:
Example links use it even weirder.. They have separate `rgb` var, but they do not use it directly for the main color, they use `hex` 😄 So honestly, I am getting lost... And I am trying to imagine users doing something like in the article examples, that they would take our `base` color and play around with opacity...
I just see 2 use cases:
- User just uses our 'final' values in his solution
- User fully changes the theme in w/e way he wants (so he changes our variables values). Now how he does that, its fully up to him, he can have his own 'base' variables and from them construct his custom colors.
But if you just want to keep those `rbg` or `hsl` values, moving to separate file makes sense to me.

Reply:
I just found a place where we use these `-rgb`/`-hsl` values internally
https://github.com/iTwin/iTwinUI/blob/888eb9ec1407af22196d914cbc9d47a0529b1ee5/packages/itwinui-css/src/style/mixins.scss#L49-L53